pkgs/top-level/all-packages.nix

```diff
@@ -16423,7 +16423,9 @@ with pkgs;
 
   knightos-mktiupgrade = callPackage ../development/tools/knightos/mktiupgrade { };
 
-  knightos-scas = callPackage ../development/tools/knightos/scas { };
+  knightos-scas = callPackage ../development/tools/knightos/scas {
+    stdenv = if stdenv.cc.isClang then llvmPackages_15.stdenv else stdenv;
```

Comment:
Why llvm15 ? `strcasecmp` should be related to either standard library or darwin sdk. Maybe their is a change in llvm16 in header search path